lib/iotaUtils.js

```diff
@@ -155,7 +155,7 @@ function mergeDeviceWithConfiguration(deviceData, configuration, callback) {
         } else if (!deviceData[fields[i]] && (!configuration || !configuration[confField])) {
             deviceData[fields[i]] = defaults[i];
         } else {
-            config.getLogger().error(context, 'There is no possible merge');
+            config.getLogger().debug(context, 'at field %d configuration merging logic did not merge anything', i);
```

Comment:
Is "%d" the right formatter for integer in JavaScript?

Reply:
Changing log from error to debug level, sure?

Reply:
Probably you are seeing a partical commit... if you clear filters in diff tab the actual change shown is the addition of the a debug line. I mean, there wasn't any error() logging here before.

Reply:
you're rigtht

Reply:
Confirmed "%d" is correct. Thus NTC (I guess)

Reply:
fixed in a2e11f62c9516932f9f4e42afc7a52b915322831